modules/sw360/sw360-workflow/src/test/java/org/eclipse/sw360/antenna/sw360/SW360MetaDataUpdaterTest.java

```diff
@@ -148,7 +177,6 @@ public void testGetLicensesWithExistingLicense() {
 
         assertThat(licenses).hasSize(1);
 
-        verify(licenseClientAdapter, times(1)).isLicenseOfArtifactAvailable(licenseName);
```

Comment:
You could add an assertion here, that licenseclientadpater getLicenses is only called once.

Reply:
Done